VERSION

```diff
@@ -1 +1 @@
-0.24.0-dev
+0.23.1
```

Comment:
I think this also needs to be `0.24.0-dev` because after merging this version string will be embedded in the binary, right? :smile:

Reply:
💯

Reply:
https://github.com/thanos-io/thanos/issues/4741